pkg/util/containers/metrics/provider/metacollector.go

```diff
@@ -68,6 +70,25 @@ func (mc *metaCollector) GetContainerIDForPID(pid int, cacheValidity time.Durati
 	return "", nil
 }
 
+// GetContainerIDForInode returns a container ID for given Inode.
+func (mc *metaCollector) GetContainerIDForInode(inode uint64, cacheValidity time.Duration) (string, error) {
+	mc.lock.RLock()
+	defer mc.lock.RUnlock()
+
+	for _, collectorRef := range mc.containerIDFromInodecollectors {
+		val, err := collectorRef.Collector.GetContainerIDForInode(inode, cacheValidity)
+		if err != nil {
+			return "", err
+		}
+
+		if val != "" {
```

Comment:
I think we should either document that `GetContainerIDForInode` can return an empty string without error or enforce that `GetContainerIDForInode` does not return an empty string.

Reply:
Got it, done 👍